js/src/components/SchemaTab.js

```diff
@@ -89,14 +90,14 @@ const Content = ( props ) => {
 				linkText={ __( "Learn more about page or content types", "wordpress-seo" ) }
 			/>
 			<Select
-				id={ `yoast_wpseo_schema_page_type_react_${ props.location }` }
```

Comment:
reminder for myself to check the consequences of this. I believe we added the "_react" bit to differentiate from the hidden fields